torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1405,21 +1405,22 @@ std::tuple<at::Tensor, at::Tensor> XLANativeFunctions::kthvalue(
                          bridge::AtenFromXlaTensor(std::get<1>(results)));
 }
 
-at::Tensor XLANativeFunctions::leaky_relu(const at::Tensor& self,
-                                          const at::Scalar& negative_slope) {
-  TORCH_LAZY_FN_COUNTER("xla::");
-  return bridge::AtenFromXlaTensor(tensor_methods::leaky_relu(
-      bridge::GetXlaTensor(self), negative_slope.to<double>()));
-}
-
 at::Tensor XLANativeFunctions::leaky_relu_backward(
```

Comment:
If you add `leaky_relu_backward` to the `full_codegen` section in xla_native_functions.yaml, I would expect another `leaky_relu_backward` definition in pytorch/xla/torch_xla/csrc/generated/XLANativeFunctions.cpp and therefore it should fail to build due to multiple definition. Is that correct?

Reply:
Yes, that's right. LeakyReluBackward is partially codegen'ed, so the aten bridging function won't be auto generated. We need to customized the bridging function due to the assertion. For LeakyReluBackward, only the XLA IR node is generated in LazyIr.h

Reply:
Thanks for the answer. Just to clarify and re-iterate the answer for the original question, the `leaky_relu_backward` is added to the `ir_gen` section, which only generates the IR related definitions. So the definition of `leaky_relu_backward` is still required here.